massa-execution-worker/src/interface_impl.rs

```diff
@@ -546,7 +546,7 @@ impl Interface for InterfaceImpl {
         validity_start: (u64, u8),
         validity_end: (u64, u8),
         max_gas: u64,
-        gas_price: u64,
+        raw_fee: u64,
```

Comment:
Since only the name changes, I guess we don't need to update anything except the docs in https://github.com/massalabs/massa-sc-runtime? Could you please point me in the right direction for that? @adrien-zinger @aoudiamoncef

Reply:
in massa-sc-runtime, the "gas_price" parameter needs to be renamed "raw_fee", and the documentation needs to be updated.

Here as well: https://github.com/massalabs/massa-sc-runtime/blob/46ca849403abe0f0e41da1b10c65a4d652f8b3a0/src/execution/as_abi.rs#L490

Here as well: https://github.com/massalabs/massa-sc-runtime/blob/main/src/execution/as_abi.rs#L514-L516

Here as well: https://github.com/massalabs/massa-sc-runtime/blob/main/src/execution/as_abi.rs#L527

This also needs to be updated: https://github.com/massalabs/massa-as-sdk/blob/994da7b5779a5c8714da50402be50f8332008e23/assembly/std/index.ts#L226-L266